Gemfile

```diff
@@ -1,16 +1,19 @@
 source 'https://rubygems.org'
 gemspec
 
-gem 'debugger',      '~> 1.6.6'
+gem 'byebug', '~> 3.5.1' if RUBY_VERSION > '2'
 gem 'yard',          '~> 0.8.5'
 gem 'rake',          '~> 0.9'
 gem 'awesome_print', '~> 1.1.0'
 gem 'redcarpet'
 
-gem 'representors', git: 'https://www.github.com/mdsol/crichton-representors.git', tag: 'v0.0.1.pre'
+gem 'representors', git: 'https://www.github.com/mdsol-share/representors.git', branch: '0-0-stable'
```

Comment:
The whole stable branch thing is my [Cunning Plan](https://sites.google.com/a/mdsol.com/knowledgebase/home/departments/engineering/on-demand-portfolio/paas-high-performance-teams/mcc-teams-collaboration-page/medidata-clinial-cloud/platform-services-blog/gemfilebestpractices). Note that in any given project, this resolves to a SHA in the Gemfile.lock.